packages/api-client/src/views/Request/Request.vue

```diff
@@ -93,6 +95,26 @@ const executeRequest = async () => {
   if (!activeRequest.value || !activeExample.value || !activeCollection.value)
     return
 
+  const paramTypes = ['path', 'query', 'headers', 'cookies'] as const
+  // Checks if any required parameters are empty
+  const hasEmptyRequiredParameters = paramTypes.some((paramType) => {
+    return activeExample.value?.parameters[paramType].some((param) => {
+      if (!param.enabled) return false
+
+      if (hasEmptyRequiredParameter(param)) {
+        toast(`Required ${paramType} parameter ${param.key} is empty`, 'error')
+        events.requestStatus.emit('abort')
+        return true
+      }
+      return false
+    })
+  })
+
+  if (hasEmptyRequiredParameters) {
+    events.cancelRequest.on(cancelRequest)
```

Comment:
wait what this? cancel request? We don't want to cancel

Reply:
cancels the request if a required field is empty as this would throw an error anyway?

Reply:
we don't want to cancel the request, we want to let it go

Reply:
we talked that we would let it go regarding type and value validation but thought we would prevent it for required one? cc @hanspagel

Reply:
i'll update accordingly

Reply:
nope just let em all go! use case: what if someone is testing what happens when you miss required params 😄